tools/reviewer/web/src/app/dashboard/diffs/diffs.component.ts

```diff
@@ -104,7 +105,7 @@ export class DiffsComponent implements OnInit {
           } else if (needAttentionOfList.includes(userEmail)) {
             // Need attention of user
             this.diffGroups[DiffGroups.NeedAttention].push(diff);
-            this.diffGroups[DiffGroups.Outgoing].push(diff);
+            this.diffGroups[DiffGroups.Incoming].push(diff);
```

Comment:
If should be in `Incoming` if `userEmail` is in reviewers, whether it's in `needsAttention` or not.